generic/tests/linux_stress.py

```diff
@@ -0,0 +1,208 @@
+import time
+import logging
+import aexpect
+
+from avocado.utils import download
+
+from virttest import data_dir
+from virttest import utils_misc
+from virttest import error_context
+
+
+class Stress(object):
```

Comment:
We already have `VMStress` described in `virttest/utils_test/__init__.py`, so I'd suggest to modify it instead of creating a new one.

Reply:
@luckyh I checked the VMStress, it actually use autotest run its control files, It looks modifying it will be kind of creating a new one.

Reply:
But it needs to be modified as well, doesn't it?